kubernetes/single-tenant/deploy.json

```diff
@@ -1,195 +1,273 @@
 {
-  "kind": "PersistentVolume",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "recordings",
-    "labels": {}
-  },
-  "spec": {
-    "capacity": {
-      "storage": "2Gi"
+    "kind": "PersistentVolume",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "recordings",
+        "labels": {}
     },
-    "accessModes": [
-      "ReadWriteMany"
-    ],
-    "persistentVolumeReclaimPolicy": "Retain",
-    "hostPath": {
-      "path": "/root/recordings"
+    "spec": {
+        "capacity": {
+            "storage": "2Gi"
+        },
+        "accessModes": [
+            "ReadWriteMany"
+        ],
+        "persistentVolumeReclaimPolicy": "Retain",
+        "hostPath": {
+            "path": "/root/recordings"
+        }
     }
-  }
 }
 
 {
-  "kind": "PersistentVolumeClaim",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "recordings"
-  },
-  "spec": {
-    "resources": {
-      "requests": {
-        "storage": "2Gi"
-      }
+    "kind": "PersistentVolumeClaim",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "recordings"
     },
-    "accessModes": [
-      "ReadWriteMany"
-    ]
-  }
+    "spec": {
+        "resources": {
+            "requests": {
+                "storage": "2Gi"
+            }
+        },
+        "accessModes": [
+            "ReadWriteMany"
+        ]
+    }
 }
 
 {
-  "kind": "Deployment",
-  "apiVersion": "extensions/v1beta1",
-  "metadata": {
-    "name": "vgw",
-    "creationTimestamp": null,
-    "labels": {
-      "service": "vgw"
-    }
-  },
-  "spec": {
-    "replicas": 1,
-    "selector": {
-      "matchLabels": {
-        "service": "vgw"
-      }
-    },
-    "template": {
-      "metadata": {
-        "creationTimestamp": null,
+    "kind": "Deployment",
+    "apiVersion": "extensions/v1beta1",
+    "metadata": {
+        "name": "vgw",
         "labels": {
-          "service": "vgw"
+            "service": "vgw"
         }
-      },
-      "spec": {
-        "volumes": [{
-            "persistentVolumeClaim": {
-              "claimName": "recordings"
+    },
+    "spec": {
+        "replicas": 1,
+        "selector": {
+            "matchLabels": {
+                "service": "vgw"
+            }
+        },
+        "template": {
+            "metadata": {
+                "labels": {
+                    "service": "vgw"
+                }
             },
-            "name": "recordings"
-          }],
-        "hostNetwork": true,
-        "imagePullSecrets": [{
```

Comment:
I don't know what imagePullSecrets does but why was it removed? I see a bunch of secrets related changes further down in this script as well. What does that have to do with quiesce? If it has nothing to do with quiesce please make these changes in a separate pull request in the future along with an explanation of why the changes are being made.